modules/consolidated-scanner.adoc

```diff
@@ -3,12 +3,12 @@
 // * operating/examine-images-for-vulnerabilities.adoc
 :_mod-docs-content-type: CONCEPT
 [id="about-scanner-v4_{context}"]
-= About RHACS Scanner V4
+= About the RHACS scanner
 
 [role="_abstract"]
 {product-title-short} provides its own scanner, or you can configure an integration to use {product-title-short} with another vulnerability scanner.
 
-Beginning with version 4.4, Scanner V4, built on link:https://github.com/quay/claircore[ClairCore], provides scanning for language and operating system-specific image components. Currently, {product-title-short} also uses the StackRox Scanner. To continue receiving full scanning support with the latest features, usage of both scanners is required.
+Before release 4.8, the default scanner in {product-title-short} was the StackRox Scanner. This scanner is deprecated. Scanner V4, introduced in release 4.4 and built on link:https://github.com/quay/claircore[ClairCore], provides scanning for language and operating system-specific image components and node and platform scanning.
```

Comment:
Claircore.

Also, Scanner V4 specifically only supports RHCOS. I am not 100% sure if we plan on dropping the other nodes we supported in StackRox Scanner. Similarly, the platform scanning Scanner V4 supports is the newer style (where we separated OpenShift images into a Platform section) and not quite the older definition of platform. Not sure if we want to bother calling this out or not, though